apsuite/commisslib/meas_fofb_sysid.py

```diff
@@ -266,17 +266,24 @@ def get_levels_bpms_from_svd(self, lvl0, lvl1):
 
         """
         respm = self.params.svd_levels_respmat
+        # Zeroing RF column
+        respm[:, -1].fill(0)
         singval = self.params.svd_levels_singmode_idx
 
         if self.params.svd_levels_regularize_matrix:
             respm = self._calc_matrix_regularization(respm)
         respm = self._convert_respmat_from_phys2hard_units(respm)
 
-        u, *_ = self._calc_svd(respm)
+        u, s, _ = self._calc_svd(respm)
         us = u[:, singval]
         us /= _np.abs(us).max()
+        ss = s[singval]
```

Comment:
There's a bug here. You need to normalize before selecting, otherwise `ss` will be always one.

Reply:
Thanks for the catch, @fernandohds564. 1bb8002b4c4cb7e78cd6f049024c9a0a780d172f fixes it. Are ok with that?